intercom-plugin/src/ios/IntercomBridge.m

```diff
@@ -114,8 +117,66 @@ - (void)displayHelpCenter:(CDVInvokedUrlCommand*)command {
     [self sendSuccess:command];
 }
 
-- (void)hideMessenger:(CDVInvokedUrlCommand*)command {
-    [Intercom hideMessenger];
+- (void)displayHelpCenterCollections:(CDVInvokedUrlCommand*)command {
+    NSDictionary *args = command.arguments[0];
+    NSArray* collectionIds = args[@"collectionIds"];
+    [Intercom presentHelpCenterCollections:collectionIds];
+    [self sendSuccess:command];
+}
+
+- (void)fetchHelpCenterCollections:(CDVInvokedUrlCommand*)command {
+    [Intercom fetchHelpCenterCollectionsWithCompletion:^(NSArray<ICMHelpCenterCollection *> * _Nullable collections, NSError * _Nullable error) {
+        if (error) {
+            CDVPluginResult *pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsNSInteger:error.code];
+            [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
+        } else {
+            NSMutableArray *array = [[NSMutableArray alloc] init];
+            for (int i = 0; i < collections.count; i++)
```

Comment:
Raw loops like this always make me feel a bit nervous. Like obviously this is perfectly well implemented... but they are known error hotspots. Could we use `for in` instead? 
`for (ICMHelpCenterCollection *collection in collections) {`

Reply:
Makes sense. For future reference, this one applies to Android as well! 😄